tests/aiq_tests/test_utils.py

```diff
@@ -87,6 +105,17 @@ def create_sample_analysis_response(is_false_positive: str = CVEValidationStatus
             prompt=prompt
         )
 
+    @staticmethod
+    def create_sample_tracker_from_dict(issues_dict: Dict[str, PerIssueData], config: Optional[Config] = None, 
```

Comment:
can't we modify/extend the `create_sample_tracker` instead of introducing this method?